homeassistant/components/vizio/config_flow.py

```diff
@@ -101,6 +114,25 @@ def __init__(self) -> None:
         """Initialize config flow."""
         self._user_schema = None
         self._must_show_form = None
+        self._ch_type = None
+        self._pairing_token = None
+        self._data = None
+
+    async def _create_entry_if_unique(
+        self, input_dict: Dict[str, Any]
+    ) -> Dict[str, Any]:
+        """Check if unique_id doesn't already exist. If it does, abort. If it doesn't, create entry."""
+        unique_id = await VizioAsync.get_unique_id(
```

Comment:
What happens here if access token is not valid ?

Reply:
When the access token isn't valid, config flow fails at this check here: https://github.com/raman325/home-assistant/blob/vizio_add_pairing_to_config_flow/homeassistant/components/vizio/config_flow.py#L164-L170

That only gets called if the config has been validated